extensions/binding/org.eclipse.smarthome.binding.astro/ESH-INF/thing/channels.xml

```diff
@@ -329,14 +329,14 @@
 		<item-type>Number</item-type>
 		<label>Moon age</label>
 		<description>The age of the moon in days</description>
-		<state readOnly="true" />
+		<state readOnly="true" pattern="%d d" />
 	</channel-type>
 
 	<channel-type id="illumination">
 		<item-type>Number</item-type>
 		<label>Moon illumination</label>
 		<description>The illumination of the moon in %</description>
-		<state readOnly="true" />
+		<state readOnly="true" pattern="%d %%"  />
```

Comment:
Duration is a long, moon age is integer and illumination is double.
So only illumination needs a change  to "%.0f %%".